package.json

```diff
@@ -64,8 +64,8 @@
     }
   ],
   "engines": {
-    "node": ">= 10",
-    "npm": ">= 6.14.0"
+    "node": ">= 16",
+    "npm": ">= 8.15.0"
```

Comment:
Any reason for the more specific version constraint (compared to the css package)?

Reply:
No, there wasn't a reason. Adjusted to be consistent.